requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Version updates managed by dependabot
 
-broker[docker]==0.2.9
+broker[docker]==0.2.10
```

Comment:
https://github.com/SatelliteQE/broker/pull/171/files#diff-58e4c8652c2fa8a7a66c1f81502625c019a9a979eb42dd6d711b3f0db6f0257bR6 is breaking dependencies install in GHA, as it seems `cache` isn't available for py3.8 and it was introduced in py3.9 as per https://github.com/python/cpython/pull/20019
So I would like to ask if we're planning to move away from py3.8 as CI is already using the py3.11 quay image or do you want to handle this in some way in the broker itself?
@JacobCallahan @jyejare ^

Reply:
@Gauravtalreja1 we're actually moving our pipelines to run on 3.11, so I'll be adjusting robottelo's checked versions as well. Hold on this

Reply:
@JacobCallahan Cool, thanks for the update

Reply:
We have to also see the compatibility of upgrade repository with 6.11, because updating to Python 6.11 with broker 0.2.10 will work but the upgrade still runs with Python 3.8 and hence our `upgrade-robottelo` image will have broken dependency issues on merging this PR.

I am raising a PR for upgrade to check the compatibility  with Python 3.11

Reply:
BTW, The good part is that we are checking the compatibility with 3.11 in upgrade already, but just that we have to update the image now.

Reply:
CI MR-865 is raised by @lpramuk to update upgrade-robottelo image for Python 3.11

Reply:
@JacobCallahan I raised PR #10493 for Python 6.11, 6.10 checks and removed 6.8.